com.unity.formats.fbx/Editor/ExportModelEditorWindow.cs

```diff
@@ -226,7 +226,20 @@ internal int SelectedFbxPath
             set { m_selectedFbxPath = value; }
         }
 
-        protected Object[] ToExport { get; set; }
+        private bool m_exportSetContainsPrefabInstanceWithAddedObjects;
```

Comment:
can you comment that this caches the result of SelectionContainsPrefabInstanceWithAddedObjects?